src/routes/report-route.tsx

```diff
@@ -91,6 +92,8 @@ function createReportRoute(
       reportControls.push(Control.BY_AGENCY);
     }
 
+    const title = (ALL_ROUTES as Record<string, string>)[path];
```

Comment:
Could we avoid the type-casting by assigning the `path` type on `ReportRouteProps` as `keyof typeof ALL_ROUTES` ? Or would that cause some upstream problems for how the component is rendered?

Reply:
It worked! Sort of, but the `Object.entries().map()` lost some fidelity of the keys. PTAL at c1f74d6618489982494659058b8a4030d2f7cc30, it still has a cast, but's more specific?

Reply:
Yeah that looks good to me 👍